book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] [Optional] Kindly make this attribute have both getter and setter methods. For example, `attr_accessor :rentals`. 👍